pkg/network/ebpf/c/runtime/conntrack.h

```diff
@@ -28,10 +28,15 @@ static __always_inline u32 get_netns(void *p_net) {
     return net_ns_inum;
 }
 
-static __always_inline u32 ct_status(const struct nf_conn *ct) {
-    u32 status = 0;
-    bpf_probe_read_kernel_with_telemetry(&status, sizeof(status), (void *)&ct->status);
-    return status;
+#define RETURN_IF_NOT_NAT(orig, reply)                      \
```

Comment:
Can we continue to use the status field for runtime compiled? Its less code.

Reply:
sure that makes sense, also we won't have the issues guessing for it